lib/streamlit/__init__.py

```diff
@@ -117,7 +117,7 @@
 from streamlit.util import functools_wraps as _functools_wraps
 
 # Modules that the user should have access to.
-from streamlit.caching import cache  # noqa: F401
+from streamlit.caching import cache as cache  # noqa: F401
```

Comment:
what is the reason behind `as` here?

Reply:
In `lib/mypy.ini` I set `implicit_export = false` (https://mypy.readthedocs.io/en/stable/config_file.html#miscellaneous-strictness-flags), which is documented as `By default, imported values to a module are treated as exported and mypy allows other modules to import them. When false, mypy will not re-export unless the item is imported using from-as or is included in __all__.`

Example apps were failing `mypy` checking by using `st.cache`, and this fixed that, while still having us fail if someone tried to use `st.subprocess`. We probably should define `__all__` in the future to make our interface explicit, but in the short term should I leave this change in, or instead see if I can change the value of `implicit_export` for this module only?

Reply:
I'm OK with leaving it this way for now. Just add a comment explaining the weird `as`.

Reply:
@kantuni, added comment.